cosmosdb/secure-cosmosdb-get-connection-string/secure-cosmosdb-get-connection-string

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
```

Comment:
GitHub won't let me comment on the correct line, but generated the following errors:

```
az cosmosdb show: error: argument --resource-group/-g: expected one argument
usage: az cosmosdb show [-h] [--verbose] [--debug]
                        [--output {json,jsonc,table,tsv,yaml,none}]
                        [--query JMESPATH] [--subscription _SUBSCRIPTION]
                        [--resource-group RESOURCE_GROUP_NAME]
                        [--name ACCOUNT_NAME] [--ids ID [ID ...]]                                                                                             az cosmosdb list-keys: error: argument --resource-group/-g: expected one argument
usage: az cosmosdb list-keys [-h] [--verbose] [--debug]
                             [--output {json,jsonc,table,tsv,yaml,none}]
                             [--query JMESPATH] [--subscription _SUBSCRIPTION]
                             [--resource-group RESOURCE_GROUP_NAME]
                             [--name ACCOUNT_NAME] [--ids ID [ID ...]]
```

Looks like the script still uses `$myRG` in places when the variable is `$resourceGroupName`.

Reply:
fixed in next commit.